### PR TITLE
Bump Ansible version to 2.13

### PR DESCRIPTION
**Pull request by rjeffman**
Bump ansible-freeipa collection minimum version to Ansible 2.13.

Please, see individual commits for details.

### CODE REVIEW

File: utils/ansible-freeipa.spec.in [hunk 1]
```diff
@@ -35,6 +35,7 @@ Features
 - Modules for automount key management
 - Modules for automount location management
 - Modules for automount map management
+- Modules for certificate management
```
Comment (t-woerner) [inline utils/ansible-freeipa.spec.in:38]: This should be a separate commit and at also PR.

File: utils/ansible-freeipa.spec.in [hunk 2]
```diff
@@ -49,6 +50,7 @@ Features
 - Modules for hostgroup management
 - Modules for idrange management
 - Modules for location management
+- Modules for netgroup management
```
Comment (t-woerner) [inline utils/ansible-freeipa.spec.in:53]: Same as above ^^

File: utils/ansible-freeipa.spec.in (hunk 1 above)
Comment (rjeffman) [reply utils/ansible-freeipa.spec.in]: fixed.

**Review by t-woerner — APPROVED** [review]
LGTM
